lib/cli_common/cli_common/log.py

```diff
@@ -53,26 +54,51 @@ def send(logger, method_name, event_dict):
         return event_dict
     return send
 
+def setup_papertrail():
+    """
+    Setup papertrail account using taskcluster secrets
+    """
+
+    # Load secrets from environment
+    from cli_common.taskcluster import get_secrets
+    try:
+        secrets = get_secrets(required=('PAPERTRAIL_HOST', 'PAPERTRAIL_PORT'))
+    except:
+        return
+
+    # Setup papertrail
+    app_name = os.environ.get('PROJECT', 'UnknownApp')
+    papertrail = logbook.SyslogHandler(
+        application_name=app_name,
+        address=(secrets['PAPERTRAIL_HOST'], int(secrets['PAPERTRAIL_PORT'])),
+        format_string='{record.time} {record.channel}: {record.message}',
+        bubble=True,
+    )
+    papertrail.push_application()
+
 
 def init_app(app):
     """
     Init logger from a Flask Application
     """
     mozdef = app.config.get('MOZDEF_TARGET', None)
-    level = logbook.ERROR
+    level = logbook.INFO
     if app.debug:
         level = logbook.DEBUG
     init_logger(level=level, mozdef=mozdef)
 
 
-def init_logger(level=logbook.ERROR, handler=None, mozdef=None):
+def init_logger(level=logbook.INFO, handler=None, mozdef=None):
```

Comment:
one of the arguments of ``init_logger`` should be ``app_name`` which should default to ``None``. when None look up ``APP_NAME`` in ``os.environ``.

If ``app_name`` is ``None`` even after looking up the ``os.environ`` then raise the exception. raising the exception will make sure we are logging the application properly. I rather see we fail early then *swallowing* the error.